### DISCUSSION TITLE
How to add a custom coloured two column table?

### DISCUSSION THREAD

**Discussion by Rainymood** [mmistakes/minimal-mistakes#2480] [Q&A] [answered]
What would be the easiest way of reproducing such a custom coloured two-column table as shown below? 

![Screenshot 2020-04-08 at 12 44 44](https://user-images.githubusercontent.com/2186137/78775598-c24b6e00-7996-11ea-84a9-ea973f81ae13.png)

Definitely willing to figure this out on my own just need a hint in the right direction! 

Website: https://dfrieds.com/

**Comment by mmistakes** [comment, marked as answer]
Modify the CSS to style the [`table` element and friends](https://github.com/mmistakes/minimal-mistakes/blob/master/_sass/minimal-mistakes/_tables.scss) differently.

https://mmistakes.github.io/minimal-mistakes/docs/stylesheets/#customizing

**Comment by mmistakes** [comment]
Also, that site isn't built with tables. Inspect the HTML and you'll see it's just wrapper `div`s and some unordered lists. Looks like they're using Bootstrap for the grid and their `panel` component for styling.

![image](https://user-images.githubusercontent.com/1376749/78787259-1561f880-7978-11ea-8e4b-03f55efb30f7.png)
